src/plots/barPlot.ts

```diff
@@ -291,6 +291,7 @@ export module Plots {
     }
 
     protected _visibleOnPlot(datum: any, pixelPoint: Point, selection: d3.Selection<void>): boolean {
+      Utils.Window.deprecated("v1.1.0");
```

Comment:
Possibly put a symbol here that allows Grunt to find/replace the symbol with current version number + 1.  That said, that's outside scope of this PR, but something to keep in mind especially as we are forced to deprecate more things